### PR TITLE
SYS-3296 - fix: A documentation on dependabot and codeowners.

### PR DESCRIPTION
**Pull request by Abdulgafaar**
This PR is to add documentation on the usage of CODEOWNERS and dependabot.

### CODE REVIEW

File: docs/workflows/dependabot.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Dependabot
+
+This ensures that all dependencies are kept up to date and there are no vulnerabilities. it checks for the depencency files if there are outdated requierements and opens individual PRs if it finds any for reviews.
```
Comment (hugolesta) [inline docs/workflows/dependabot.md:3]: Typo: should be "dependency" and "requirements"

File: docs/workflows/codeowners.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Codeowners
+
+This is besically a file in the root directory of the repo to define teams that are responsible for the repository.
```
Comment (hugolesta) [inline docs/workflows/codeowners.md:3]: typo: basically

**Review by hugolesta — CHANGES_REQUESTED** [review]

**Comment by Abdulgafaar** [comment]
I have updated the doc based on your reviews.

File: docs/dependabot.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
+# Dependabot
+
+This ensures that all dependencies are kept up to date and there are no vulnerabilities. it checks for the dependency files if there are outdated requirements and opens individual PRs if it finds any for reviews.
```
Comment (mattialacroce) [inline docs/dependabot.md:3]: "There are no vulnerabilities" is not exactly true ahhahah but it's okay!

**Review by mattialacroce — APPROVED** [review]

**Review by hugolesta — APPROVED** [review]

**Pull request merged**